ci-operator/step-registry/sandboxed-containers-operator/env-cm/sandboxed-containers-operator-env-cm-commands.sh

```
@@ -82,6 +82,31 @@ mirror_konflux() {
   oc apply -f "https://raw.githubusercontent.com/openshift/trustee-fbc/refs/heads/main/.tekton/images-mirror-set.yaml"
 }
 
+wait_for_catsrc() {
+  local catsrc_name="$1"
+  echo ">>> Waiting for CatalogSource ${catsrc_name} to be READY..."
+
+  local ready=false
+  for i in {1..24}; do
+    local state=$(oc get catalogsource -n openshift-marketplace "${catsrc_name}" -o jsonpath='{.status.connectionState.lastObservedState}' 2>/dev/null || echo "")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)